src/stores/wordpress-versions-slice.ts

```diff
@@ -112,19 +112,17 @@ const wordpressVersionsSlice = createSlice( {
 	selectors: {
 		selectWordPressVersions: ( state ) => state.versions,
 		selectWordPressVersionsWithLatest: ( state ) => {
-			const latestNonBeta = state.versions.find( ( version ) => ! version.isBeta );
-			if ( ! latestNonBeta ) {
-				return state.versions;
-			}
-			const otherVersions = state.versions.filter( ( version ) => version !== latestNonBeta );
-			return [
-				{
-					isBeta: false,
-					label: `${ latestNonBeta.label } (${ __( 'latest' ) })`,
-					value: latestNonBeta.value,
-				},
-				...otherVersions,
-			];
+			let foundLatest = false;
```

Comment:
```suggestion
			let foundLatestStable = false;
```
Nit, but this would clarify what we're looking for.